packages/EasyCodingStandard/composer.json

```diff
@@ -51,6 +51,9 @@
             "Symplify\\EasyCodingStandard\\Tests\\": "tests"
         }
     },
+    "replace": {
+        "symfony/polyfill-php70": "*"
+    },
```

Comment:
This prevents https://github.com/symfony/polyfill/issues/191#issuecomment-573092820